### DISCUSSION TITLE
questions about coordinate and dz

### DISCUSSION THREAD

**Discussion by wwwliye** [NCAR/icar#191] [Q&A]
Hi~ I want to know how to modify the model top height,   and in icar_options.nml, what's the usage of "dz_levels" ,could I modify the number of "dz_levels" and if could, according to what to modify, thank you for your reply~

**Comment by gutmann** [comment]
You can change the number of vertical levels and the thickness of those levels to set the model top using 
nz https://github.com/NCAR/icar/blob/9ee76831e6fa432f4ee185ecbe1aadb600eee0a4/run/complete_icar_options.nml#L194
and the dz_levels variable you were looking at.
